edit_course/operations/clone.py

```diff
@@ -81,6 +112,9 @@ def clone(
     if clone_assistants:
         instance.set_assistants(assistants)
 
+    if siskey != 'none':
```

Comment:
```suggestion
    if siskey and siskey != 'none':
```
We could also allow an empty `siskey` that would mean that no SIS setup is made like with `'none'`.